<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgColorbarColoring</class>
 <widget class="QDialog" name="DlgColorbarColoring">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>color by colorbar</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0">
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QRadioButton" name="radBar">
       <property name="text">
        <string>color bar</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="JKQTPMathImageColorPaletteComboBox" name="cmbColorbar"/>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="radBar2">
       <property name="text">
        <string>2-color bar:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="ColorComboBox" name="cmbCol21">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="ColorComboBox" name="cmbCol22">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="radBar3">
       <property name="text">
        <string>3-color bar:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="ColorComboBox" name="cmbCol31">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="ColorComboBox" name="cmbCol32">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="ColorComboBox" name="cmbCol33">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorComboBox</class>
   <extends>QComboBox</extends>
   <header>colorcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>JKQTPMathImageColorPaletteComboBox</class>
   <extends>QComboBox</extends>
   <header>jkqtpimageelements.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgColorbarColoring</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>620</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgColorbarColoring</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>620</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radBar</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbColorbar</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>449</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radBar2</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbCol21</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>472</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>473</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radBar2</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbCol22</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>479</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radBar3</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbCol31</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>526</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>526</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radBar3</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbCol32</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>552</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radBar3</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbCol33</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>530</y>
    </hint>
    <hint type="destinationlabel">
     <x>128</x>
     <y>579</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
